mopidy_alsamixer/mixer.py

```diff
@@ -70,13 +74,20 @@ def get_volume(self):
         if not channels:
             return None
         elif channels.count(channels[0]) == len(channels):
-            return int(channels[0])
+            unadjusted_volume = channels[0]
+            if self.logarithmic_volume:
+                unadjusted_volume = 100.0 * math.pow(unadjusted_volume / 100.0, 3.0)
```

Comment:
I have pushed a commit using the GstAudio convert_volume function for cubic b12e38bffa6c3727c2c8e1a40ef91e2ea799cac8. That seems to work.

I have also pushed a commit adding logarithmic volume control ad82d9bda7ca6ebc6841c5b1d449aaa819dbad66 but I could not use the convert_volume function as it outputs DB (e.g. negative numbers etc) and we currently use a percentage to control the mixer volume. So I've taken the formula from the GstAudio page (and modified the scaling factor as per my graph above) to give a logarithmic scale, albeit with the formula in our class.

I don't think the logarithmic volume scale is as good as the cubic one! Well, at least it doesn't match up to the alsamixer volume scale like the cubic one does. Still, the cubic one is better!